src/card.css

```
@@ -0,0 +1,92 @@
+.card {
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  font-family: "Roboto", sans-serif;
+  display: flex;
+  flex-direction: column;
+}
+
+.card__logo {
+  width: 160px;
+  height: 134px;
+  margin: 32px auto 40px;
+}
+
+.card__content {
+  margin: 0 16px;
+}
+
+.card__header {
+  font-size: 12px;
+  font-weight: 500;
+  color: #060b35;
+  line-height: 18px;
+}
+
+.card__code {
+  font-weight: 400;
+  font-size: 10px;
+  color: #616070;
+  line-height: 14px;
+  margin-top: 4px;
+}
+
+.card__stars {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin-top: 16px;
+}
+
+.card__reviews {
+  font-weight: 400;
+  font-size: 10px;
+  color: #060b35;
+
+}
+
+.card__price {
+  margin-top: 23px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.card__price-value {
+  font-weight: 400;
+  font-size: 12px;
+  color: #616070;
+}
+
+.card__price-cost {
+  font-weight: 700;
+  font-size: 16px;
+  color: #060b35;
+}
+
+.card__button {
+  box-sizing: border-box;
+  border: 1px solid #00acdc;
+  text-decoration: none;
+  background: #00acdc;
+  color: white;
+  border-radius: 5px;
+  height: 40px;
+  width: 100%;
+  font-size: 14px;
+  font-weight: 700;
+  text-align: center;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-transform: uppercase;
+  transition: color 0.8s linear;
```

У тебе ще змінюється background-color
```suggestion
  transition: color 0.8s linear, background-color 0.8s;
```